benchmark/strong_scaling_shallow_water_model_single.jl

```diff
@@ -15,31 +15,35 @@ MPI.Init()
 local_rank = MPI.Comm_rank(comm)
          R = MPI.Comm_size(comm)
 
-Nx = parse(Int, ARGS[1])
-Ny = parse(Int, ARGS[2])
+decomposition = ARGS[1]
```

Comment:
It occurs to me that there is a lot of overlap between the `incompressible` and `shallow_water` model versions and they could probably be combined, but I don't think we want to do that here.  In the future we can think about how we might dipatch between different models, but not today!

Reply:
Yeah they're like 90%+ similar...

I've thought about merging them a bit but feels like we might end up with an overly general (and hard to modify) script.

But would be good to think about merging them in the future.

Reply:
I agree that this is a thougth for the future, but wanted to mention it here so we can mull it over as the code evolves